openrr-command/src/robot_command.rs

```diff
@@ -490,4 +511,24 @@ mod test {
         assert_eq!(val.0, 0);
         assert_eq!(val.1, 2);
     }
+
+    #[test]
+    fn test_filter_escape_double_quote() {
+        // This situation assumes that `command_name hoge/fuga "foo bar"` has been entered.
```

Comment:
Please avoid `hoge`, `fuga` which might be Japanese.
And add one more test case with more than three words in the quotes (a more realistic case.)

For example, `command -f "{ type: Message, value: 1.0 }"`